### DISCUSSION TITLE
Changing EventSource time interval

### DISCUSSION THREAD

**Discussion by thepeopleseason** [progrium/topframe#9] [Q&A] [answered]
In `stocks`, you have the following comment: 

> Every line of STDOUT is a new event. If the script exits, the EventSource
> closes, but by default will retry every 3 seconds. This gets you free 
> polling. However, it also works as you'd expect with long running scripts.

How do you set the interval for EventSource polling?

**Comment by progrium** [comment, marked as answer]
it's builtin to the browser. if you need something more realtime, use a persistent connection. if you need something less realtime, make your own fetch requests. hope that helps!
